test/regression.uts

```diff
@@ -4494,3 +4494,25 @@ assert bpkt.chksum == 0xf0bc and bpkt.payload.chksum == 0xbb17
 pkt = IP(len=42, ihl=6, options=[IPOption_RR()]) / UDP() / ("A" * 10)
 bpkt = IP(str(pkt))
 assert bpkt.chksum == 0xf0bc and bpkt.payload.chksum == 0xbb17
+
+= Layer binding
+
++ Test DestMACField & DestIPField
```

Comment:
For some reason, the HTML output is empty due to '+'. Using '*' instead fixes the issue.